login/index.html

```diff
@@ -8,30 +8,43 @@
 </head>
 <body>
     <header>
-        <a class="home-logo" href="/"><img class="header-img" src=../imgs/signup_logo.png></a>
+        <a class="home-logo" href="/"><img class="header-img" src=../imgs/signup_logo.png alt="홈으로"></a>
     </header>
     <main>
         <div class="container">
-            <form>
-                <label for="email">이메일</label>
+            <div class="pseudo-form">
```

Comment:
여기는 가짜 폼태그보다 진짜 폼태그를 써주는게 어떨까요? 

```suggestion
            <form class="signup-form">
```